.github/actions/docker-publish/action.yml

```
@@ -0,0 +1,73 @@
+name: "Docker Publish"
+description: "Builds and publishes a Docker image for the Ganache repo."
+inputs:
+  VERSION:
+    description: "The release version to be tagged to the Docker image, e.g. vX.x.x"
```

`vX.x.x` would be incorrect and result in a docker tag with `vvX.x.x` getting published. I think it should be just `X.x.x`.
```suggestion
    description: "The release version to be tagged to the Docker image, e.g., 4.0.1"
```

p.s., e.g., means "for example"; I think the example value should be a valid example.